packages/consensus/src/validation/block.cairo

```diff
@@ -93,3 +96,23 @@ pub fn compute_and_validate_tx_data(
 
     Result::Ok((total_fee, merkle_root(ref txids), wtxid_root))
 }
+
+
+pub fn validate_bip30_block_hash(block_height: u32, block_hash: @Digest) -> Result<(), ByteArray> {
+    if block_height == 91722 {
+        let expected_hash: Digest = hex_to_hash_rev(
```

Comment:
`hex` module is for tests only (it's inefficient)
Use `DigestTrait::new` instead: you'd need to reverse the hex and split into 8 words, 4 bytes each